samples/bluetooth/peripheral_mds/prj.conf

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+CONFIG_BT=y
+CONFIG_BT_SMP=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="Nordic_Memfault"
+CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_BAS=y
+CONFIG_BT_MDS=y
+
+CONFIG_BT_CTLR_PHY_2M=y
+CONFIG_BT_BUF_ACL_RX_SIZE=251
```

Comment:
Does the RX buffer need to be this long?

Reply:
Yes if you want to set data length to 251 then you need to increase also RX buffer